ch10-git-internals.asc

```diff
@@ -31,11 +29,11 @@ include::book/10-git-internals/sections/maintenance.asc[]
 
 include::book/10-git-internals/sections/environment.asc[]
 
-=== Summary
+=== Özet
 
-At this point, you should have a pretty good understanding of what Git does in the background and, to some degree, how it's implemented.
-This chapter has covered a number of plumbing commands -- commands that are lower level and simpler than the porcelain commands you've learned about in the rest of the book.
-Understanding how Git works at a lower level should make it easier to understand why it's doing what it's doing and also to write your own tools and helper scripts to make your specific workflow work for you.
+Bu noktada, Git'in arka planda ne yaptığını ve bir dereceye kadar nasıl uygulandığını oldukça iyi anlamış olmalısınız.
+Bu bölüm, bir dizi sıhhi tesisat komutunu kapsıyor - kitapların geri kalanında öğrendiğiniz porselen komutlardan daha basit ve daha basit olan komutlar.
```

Comment:
Buradaki "plumbing" kelimesinin tam olarak "sıhhi tesisat" olduğudan emin değilim. Daha ziyade oldukça düşük seviyeli komutlar olduğunu belirtmek için kullanılmış bir terim gibi. Şimdilik böyle kalsın, ilerde gerekirse güncelleriz. 
Alternatiflere açığım ;)